cvat-ui/src/utils/opencv-wrapper/opencv-wrapper.ts

```diff
@@ -101,7 +101,7 @@ export class OpenCVWrapper {
 
         const global = window as any;
 
-        this.cv = await global.cv;
+        this.cv = global.cv;
```

Comment:
Could you please explain this change?

Reply:
It seems new build of opencvjs `4.8.0` sets global `cv` variable not as promise that resolves `cv`, but `cv` immiditely. A change on their side, I suppose.

Reply:
Okay, I see